src/core/router/transports/ssu/packet.h

```diff
@@ -59,7 +59,10 @@ enum struct SSUSize : std::uint16_t {
   IV = 16,
   IntroKey = 32,
   BufferMargin = IV + 2,  // IV + 2 bytes size are appended on validation
-  RawPacketBuffer = MTUv4 + BufferMargin,
+  RawPacketBuffer = (MTUv4 > MTUv6 ? MTUv4 : MTUv6) + BufferMargin,
+  FragmentBuffer =
+      (PacketMaxIPv4 > PacketMaxIPv6 ? PacketMaxIPv4 : PacketMaxIPv6)
+      + BufferMargin,
```

Comment:
>(MTUv4 > MTUv6 ? MTUv4 : MTUv6)
>(PacketMaxIPv4 > PacketMaxIPv6 ? PacketMaxIPv4 : PacketMaxIPv6)

Though an odd thing to do (IMHO), this will ensure a smoother #408.